sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -557,14 +558,175 @@ public async Task ValidateMergeCertificate()
             CollectionAssert.AreEqual(mergedServerCertificate.Cer, completedServerCertificate.Cer);
         }
 
-        // Backup Restore
-        // GetCertificates
-        // GetCertificatsIncludePending
-        // GetCertificateVersions
-        // GetDeletedCertificates
-        // GetUpdatePolicy
-        // IssuerCrud
-        // ContactsCrud
+        [Test]
+        public async Task VerifyGetIssuer()
+        {
+            string issuerName = Recording.GenerateId();
+
+            string providerName = "SSLAdmin";
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName, providerName);
+
+            RegisterForCleanupIssuer(issuerName);
+
+            await Client.CreateIssuerAsync(issuer);
+
+            CertificateIssuer getIssuer = await Client.GetIssuerAsync(issuerName);
+
+            Assert.NotNull(getIssuer);
+            Assert.NotNull(getIssuer.Id);
+            Assert.AreEqual(issuer.Provider, getIssuer.Provider);
+            //Assert.AreEqual(issuer.Name, getIssuer.Name);
```

Comment:
### Issue3

**Descrption:**
The return value of issuer name always be null.

**Reproduce:**
```
private static async Task GetProviders(CertificateClient client)
        {
            await foreach (var item in client.GetPropertiesOfIssuersAsync())
            {
                Assert.IsNull(item.Name); // It will pass since Name is null.
            }

	 var getIssuer = await client.GetIssuerAsync("test");
	 Assert.IsNull(getIssuer.Value.Name); // It will pass since Name is null.
        }
```